portfolio/src/main/webapp/index.html

```diff
@@ -1,12 +1,15 @@
 <!DOCTYPE html>
 <html>
+  
   <head>
     <meta charset="UTF-8">
     <title>Sterly Deracy's Portfolio</title>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC9FzxvwpzM9EFYxz0v6MwfSf88r4JIL2E"></script>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload = "showRandomGreets();">
+  
+  <body onload = "showRandomGreets(); createMap();">
```

Comment:
Let's create a single method to call on load here, which will call both showRandomGreets() and createMap()

Reply:
Good idea! Let me know what you think of my implementation.